docker-compose/community-docker-compose.yml

```diff
@@ -10,7 +10,7 @@ version: "2"
 
 services:
     alfresco:
-        image: alfresco/alfresco-content-repository-community:7.0.0-A20
+        image: alfresco/alfresco-content-repository-community:7.0.0-A27c
```

Comment:
The acs-community-packaging build has not been pushing images to DockerHub for a month. It took three goes to work out why, hence the 'c' suffix to the tag. Enterprise's tag is simply 7.0.0-A27.